File: C++/Math/get_percentage_of_number.cpp
#include <iostream>

// Get the percentage of a given number.
float getPercentageOfNumber(float number, float percentage) {
   return number / 100 * percentage;
}

int main() {
   // Get the percentage of a given number.
   printf("%f", getPercentageOfNumber(250, 15));
}

File: Go/Math/get-percentage-of-number.go
package main

import (
	"fmt"
)

func main() {
	// Get the percentage of a given number.
	fmt.Println(getPercentageOfNumber(250, 15))
}

// Get the percentage of a given number.
func getPercentageOfNumber(number float64, percentage float64) float64 {
	return number / 100 * percentage
}
